config/crd/gateway-operator/kustomization.yaml

```diff
@@ -24,4 +24,5 @@ resources:
   - configuration.konghq.com_kongupstreams.yaml
   - configuration.konghq.com_kongvaults.yaml
   - konnect.konghq.com_konnectapiauthconfigurations.yaml
-  - konnect.konghq.com_konnectgatewaycontrolplanes.yaml
\ No newline at end of file
+  - konnect.konghq.com_konnectgatewaycontrolplanes.yaml
+  - konnect.konghq.com_konnectnetworks.yaml
```

Comment:
```suggestion
  - konnect.konghq.com_konnectnetworks.yaml

```

Reply:
This is auto generated by `make generate`.